airflow/contrib/hooks/qubole_hook.py

```diff
@@ -200,6 +203,30 @@ def get_jobs_id(self, ti):
             cmd_id = ti.xcom_pull(key="qbol_cmd_id", task_ids=self.task_id)
         Command.get_jobs_id(self.cls, cmd_id)
 
+    def get_extra_links(self, task, dttm):
```

Comment:
docstring

Reply:
This is an implementation of https://github.com/apache/airflow/pull/3533/files#diff-664e0133fa172b6e0060f2dd7b2c3504R2948, which does have a docstring. Is that not sufficient?